packages/driver/cypress/fixtures/auth/idp.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
This s a faux idp, it will generate a 'token' based on a jwt and send it back as a search param in the provide or hard coded url